src/Sylius/Bundle/WebBundle/Resources/views/Backend/Inventory/macros.html.twig

```diff
@@ -72,7 +72,7 @@
                                     <td>
                                         <ul>
                                         {% for option in variant.options %}
-                                            <li><strong>{{ option.name }}</strong>: {{ option.value }}</li>
+                                            <li><strong>{{ option.presentation }}</strong>: {{ option.value }}</li>
```

Comment:
Yeah, that's confusing... Maybe to highlight it's not property we can explicitly call `option.getPresentation` here, wdyt?